Gemeinsamer Bibliotheksverbund ISBN.js

```diff
@@ -0,0 +1,219 @@
+					}
+				],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "import",
+		"input": "{ \"ISBN\" : \"3-86688-240-8\" }",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "Bilinguale Lexik nicht materieller lexikalischer Transfer als Folge der aktuellen russisch-deutschen Zweisprachigkeit",
+				"creators": [
+					{
+						"firstName": "Katrin",
+						"lastName": "Karl",
+						"creatorType": "author"
+					}
+				],
+				"date": "2012",
+				"ISBN": "9783866882416",
+				"callNumber": "400",
+				"numPages": "683",
+				"place": "München",
+				"publisher": "Verl. Otto Sagner",
+				"series": "Slavolinguistica",
+				"seriesNumber": "15",
+				"attachments": [],
+				"tags": [
+					"CD-ROM",
+					"CD-ROM",
+					"Deutsch",
+					"Deutsch",
+					"Deutsch",
+					"Online-Publikation",
+					"Online-Publikation",
+					"Russisch",
+					"Russisch",
+					"Russisch",
+					"Wortschatz",
+					"Wortschatz",
+					"Wortschatz",
+					"Zweisprachigkeit",
+					"Zweisprachigkeit",
+					"Zweisprachigkeit"
+				],
+				"notes": [
+					{
+						"note": "Was ist das Besondere an bilingualer Lexik? Während Entlehnungen und Code-Switching in der bisherigen Forschung breit untersucht wurden, widmet sich die vorliegende Arbeit einem Desiderat und untersucht ausführlich die theoretische und empirische Natur nicht materieller lexikalischer Transfers (u.a. auch bekannt als Calques) bezogen auf die aktuelle Kontaktsituation des Russischen mit dem Deutschen. Im Theorieteil der Arbeit erfolgt eine Abgrenzung möglicher Kontaktphänomene und eine Subklassifizierung der nicht materiellen lexikalischen Transfers in insgesamt fünf Kategorien. Im empirischen Teil werden ein umfangreiches bilinguales Korpus, bestehend aus spontaner mündlicher Kommunikation sowie schriftlicher Sprachproduktion, ausgewertet und Ergebnisse einer computerunterstützt durchgeführten Bewertungsaufgabe präsentiert. Wo in dem einen Teil die realen Vorkommen im Vordergrund stehen, ist es in dem anderen Teil die Bewertung von nicht materiellen lexikalischen Transfers wie otkladyvat' na dlinnuju skamejku 'auf die lange Bank schieben`, wobei die Urteile Bilingualer mit denjenigen einer monolingualen Kontrollgruppe verglichen werden"
+					}
+				],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "import",
+		"input": "{ \"ISBN\" : \"978-3-446-44268-9\" }",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "Grundgebiete der Elektrotechnik Band 3: Aufgaben",
```

Comment:
There are several parts here:
 * `Grundgebiete der Elektrotechnik` : title of the multi-volumes
 * `Band` : German term for volume
 * `3` : volume
 * `Aufgaben` : special subtitle of this volume

How should this information be saved in Zotero? I guess that ideally we would like to have the "Band" translated into the language a citation is in...